pkg/k8s/apps/translate.go

```diff
@@ -111,6 +111,11 @@ func (tr *Translation) DevModeOff() error {
 	tr.App.SetReplicas(getPreviousAppReplicas(tr.App))
 	delete(tr.App.ObjectMeta().Annotations, model.AppReplicasAnnotation)
 
+	delete(tr.App.ObjectMeta().Annotations, model.OktetoStignoreAnnotation)
```

Comment:
I would include a comment in the code to explain why these are being removed explicitly (more or less the same comment you have put in this review)